index.html

```diff
@@ -64,5 +64,183 @@ <h2 class="container job-title">I am a Software developer </h2>
+            </div>
+        </div>
+    </div>
+ <!-- Fourth-Profesional-Art-Printing-Data -->
+ <div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+            <button class="project-button" type="button"  id="fourthTitle">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!-- Fifth-Profesional-Art-Printing-Data -->
+<div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+        <div>
+    
+            <button class="project-button" type="button" id="fifthTitle">See Project</button></div>
+    
+        </div>
+    </div>
+</div>
+
+<!-- Sixth-Profesional-Art-Printing-Data -->
+<div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+        <div>
+            
+            <button class="project-button" type="button" id="sixthTitle">See Project</button></div>
+        
+        </div>
+    </div>
+</div>
+
+
+</section><!-- #Recent Works -->
+<!-- ======================
+    About me
+    ====================== -->
+    <section class="about-me" id="about-me">
+        <div class="about-text">
+            <h2>About Me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button type="button">Get My Resume</button>
+            <hr class="hr">
+        </div>
+        
+            <div class="skill-set">
+                
+                <ul>
+                    <li class="languages">
+                        <div class="language">
+                        <img src="assets/Rectangle.png" alt="language vector" class="vector">
```

Comment:
- Please set the proper path of the image, or include the image because it's not in your `assets folder` as it is in the Figma design. Check the `Languages, frameworks and skill` images.
- Please use the proper font family for the `Language, Framework, Skills` which is `InaiMathi` as in the Figma design. Please check all text, so you use the proper font family and size for them. 

##### Figma design
![Screenshot from 2022-06-04 10-04-10](https://user-images.githubusercontent.com/58771507/171992494-d28cfa5b-7ab8-42d5-a9c8-dca5d2294648.png)

##### Your design
![Screenshot from 2022-06-04 10-01-50](https://user-images.githubusercontent.com/58771507/171992473-6c891469-ce08-4fd6-bd73-5982a62e2b3d.png)